events/voiceStateUpdate.js

```diff
@@ -92,6 +92,8 @@ module.exports = {
 				}
 				// Avoid pauseTimeout if 24/7 is enabled
 				if (await data.guild.get(player.guildId, 'settings.stay.enabled')) return;
+				// Ensure that the bot does not set pauseTimeout if timeout already exists
+				if (player.timeout) return;
```

Comment:
It is possible to put `player.timeout` and `player.pauseTimeout` checks in a single if statement. I will leave it at that for now.